CCDB/src/CcdbApi.cxx

```diff
@@ -1052,8 +1052,8 @@ void* CcdbApi::retrieveFromTFile(std::type_info const& tinfo, std::string const&
   string fullUrl = getFullUrlForRetrieval(curl_handle, path, metadata, timestamp); // todo check if function still works correctly in case mInSnapshotMode
   // if we are in snapshot mode we can simply open the file; extract the object and return
   if (mInSnapshotMode) {
-    return extractFromLocalFile(fullUrl, tinfo, headers);
     logReading(path, timestamp, headers, "retrieve from snapshot");
+    return extractFromLocalFile(fullUrl, tinfo, headers);
```

Comment:
Thanks! But since everywhere the log is done after the reading, I think it is better to have
```
auto res = extractFromLocalFile(fullUrl, tinfo, headers);
if (res) {
  logReading(path, timestamp, headers, "retrieve from snapshot");
}
return res;
```